pipeline/functional_assays/parse_functional_assay_data.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
+
+"""
+Description:
+    Parses a .tsv source file of functional assays into a list of variants
+"""
+import argparse
+import pandas as pd
+import numpy as np
+import tempfile
+import os
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-i', '--input', help='Input functional assay file.')
+    parser.add_argument('-o', '--output', help='Ouput tsv file result.')
+    options = parser.parse_args()
+    return options
+
+def extract_relevant_data_for_processing(f_in_data_frame):
+
+    new_columns = [
+        'chr',
+        'pos_hg19',
+        'ref',
+        'alt',
+        'gene_symbol',
+        'reference_sequence',
+        'hgvs_nucleotide',
+        'log_rna_depletion',
+        'functional_enrichment_score']
+
+    new_data_frame = pd.DataFrame(columns=new_columns)
+
+    new_data_frame = new_data_frame.reset_index(drop=True)
+    f_in_data_frame = f_in_data_frame.reset_index(drop=True)
+
+    new_data_frame.chr = f_in_data_frame['chromosome']
```

Comment:
you are probably looking for pandas `rename` function. Sth along the lines of `f_in_data_frame.rename(columns={'chromosome' : 'chr', 'position (hg19)' : 'pos_hg19' etc })`

Reply:
👍